### ISSUE TITLE
¿Cómo van a aprender los alumnos con el juego?

### ISSUE THREAD

**Issue by Carmenalbl** [Sputnikomk2/ProyectoSputniko#42]
[HU-1] De qué manera vamos a configurar la lógica del juego para que el fin sea que aprendan.

**Comment by Patacabra** [comment]
https://github.com/Sputnikomk2/ProyectoSputniko/blob/newferlevel2/doc/%2342.md

**Comment by Patacabra** [comment]
El juego se dividirá en al menos dos niveles diferentes.
El primer nivel será un quizz.
El segundo nivel será un minijuego.
El juego mantendrá una estética simple a la par que un uso sencillo.
El juego tendrá un sistema de puntuación, cuyos puntos serán intercambiables por alguno de los artículos disponibles de la cafetería del colegio.
Gracias a este sistema de recompensa los alumnos estarán motivados y aprenderán para poder mejorar sus marcas.

**Comment by JJ** [comment]
Tampoco lo habéis relacionado con ninguna historia de usuario. Además, ¿te parece que esto sea del Milestone 0?

Por otro lado, ninguno de los milestones tiene el texto correspondiente. Al final del nivel 1 deberíais haber trasladado el contenido del fichero markdown a la descripción del milestone, como documento de trabajo.

**Comment by Patacabra** [comment]
@JJ Corregida la relación con la historia de usuario y el milestone correspondiente a la issue.

Con respecto al fichero markdown en la descripción del milestone, se refiere a colocar el texto de la historia de usuario correspondiente al milestone?

**Comment by JJ** [comment]
> @JJ Corregida la relación con la historia de usuario y el milestone correspondiente a la issue.
> 
> Con respecto al fichero markdown en la descripción del milestone, se refiere a colocar el texto de la historia de usuario correspondiente al milestone?

No, el del milestone. Un milestone es un producto mínimamente viable, debéis describir el mismo en donde lo tengáis definido en GitHub.

**Comment by Patacabra** [comment]
Dicho y hecho, el milestone 0 ya tiene su descripción correspondiente.

Si está bien procedo a añadir la descripción de los siguientes.

**Comment by JJ** [comment]
Sólo el 1, el resto ya se verá.
